# SetCameraArrayInfo
# Service to set the camera array info

CameraArrayInfo info
---
